honeybee_energy/load/ventilation.py

```diff
@@ -63,7 +63,7 @@ class Ventilation(_LoadBase):
         * schedule
     """
     __slots__ = ('_flow_per_person', '_flow_per_area', '_flow_per_zone',
-                 '_air_changes_per_hour', '_schedule')
+                 '_air_changes_per_hour', '_schedule', '_user_data')
```

Comment:
Remove from `__slots__` since it's already in the parent class.

Reply:
commit: loads cleaned up